packages/vouching/contracts/Vouching.sol

```diff
@@ -4,7 +4,6 @@ pragma solidity ^0.4.24;
 import "openzeppelin-zos/contracts/token/ERC20/ERC20.sol";
 import "openzeppelin-zos/contracts/token/ERC20/SafeERC20.sol";
 import "openzeppelin-zos/contracts/math/SafeMath.sol";
-import "zos-lib/contracts/Initializable.sol";
```

Comment:
Can we add the explicit `Initializable` import, instead of relying on it being imported via ERC20?